Bangazon/Managers/ProductManager.cs

```diff
@@ -1,25 +1,65 @@
+using System;
 using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Data.Sqlite;
 //Written by:Chaz Henricks
 namespace BangazonCLI
 {
     // Manages Product related methods
     public class ProductManager
     {
+        private List<Product> _products = new List<Product>();
         private dbUtilities _db;
         // Constructor method to establish a connection with the database, database conenction is passed in as an argument..
         public ProductManager(dbUtilities db)
         {
             _db = db;
         }
         // Adds a new Product--passed in as an argument--to the database
-        public int AddNewProduct(Product newProduct){
-            return 4;
+        public int AddNewProduct(Product newProduct, int customer, int productType){
```

Comment:
Once we start working on ticket 4, we should skinny this down so that AddNewProduct accepts just one argument - a Product with the customerID and ProductTypeID attached.